ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/operations/SDivOperation.java

```diff
@@ -33,11 +38,21 @@ public Gas cost(final MessageFrame frame) {
 
   @Override
   public void execute(final MessageFrame frame) {
```

Comment:
Seems a lot of changes in this method. Do we have sufficient tests to ensure it is not broken ?

Reply:
Reference tests should cover it. Who would know otherwise?